.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 php:
  - hhvm
- - hhvm-nightly
 matrix:
   allow_failures:
   - php: hhvm
```

Comment:
This is currently actually failing - Travis is only green because every tested version has allowed failures